<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>380</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>海派腔调摩擦简</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
     </font>
    </property>
    <property name="text">
     <string>开始连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bujiao">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
     </font>
    </property>
    <property name="text">
     <string>不叫</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_jiao">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
     </font>
    </property>
    <property name="text">
     <string>叫地主</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buchu">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>不出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_chupai">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>出牌</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_again">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>再来一局</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
